### PR TITLE
Display sde

### PR DESCRIPTION
**Pull request by lukoktonos**
This PR adds the UI support for hybrid measures in order to display "Supplemental Data Elements" in their own section on the measure view page, with coverage/highlighting enabled for them.  It also makes it so that the results of SDEs are displayed in excel export, rather than 'NA' as was previously shown.

It also fixes a bug in the fixture export script that was saving a different version of a valueset if there were multiple valuesets with the same oid available at time of export.  Now the export saves the one with the version corresponding to the measure being exported.  This fixes a few 'WARNING: missing value set' message that was appearing when using the fixtures.

Note: the measure on bonnie-fixtures@mitre.org will need to be updated after deployment, because it is missing 'supplemental_data_elements' now.  The patient won't need to be.

Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1444
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
(Only 1 whitespace improvement that should not affect readability)

- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Test fixtures updated and documented as necessary ( see [internal wiki](https://gitlab.mitre.org/bonnie/internal-documentation/wikis/testing#test-fixtures) )
- [x] Code coverage has not gone down and all code touched or added is covered. 

Branch | Back End Coverage | Front End Coverage
-- | -- | --
bonnie-v2.2 | 1746 / 1964 LOC (88.9%) covered. | image
display_sde | 1746 / 1964 LOC (88.9%) | image

- [x] Automated regression test(s) pass **all results equal**

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA test links: N/A
- [x] Justification for using JIRA tests: N/A
- [x] JIRA tests have been added to sprint N/A

**Reviewer 1:**

Name: @mayerm94 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass N/A
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A

**Reviewer 2:**

Name: @c-monkey 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass N/A
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A

### CODE REVIEW

**Review by c-monkey — APPROVED** [review]
Looks good

File: app/assets/javascripts/helpers/cql_results_helpers.js.coffee [hunk 1]
```diff
@@ -60,6 +60,11 @@ class CQLResultsHelpers
       for statementName of statements
         statementRelevance[lib][statementName] = "NA"
 
+    if CQLMeasureHelpers.isHybridMeasure(measure)
+      for statement in populationSet.get('supplemental_data_elements')
+        # Mark all Supplemental Data Elements as relevant
+        @_markStatementRelevant(measure.get('cql_statement_dependencies'), statementRelevance, measure.get('main_cql_library'), statement, "TRUE")
```
Comment (mayerm94) [inline app/assets/javascripts/helpers/cql_results_helpers.js.coffee:66]: Can we try to mark SDEs from all libraries instead of just the main_cql_library?

Comment (mayerm94) [reply app/assets/javascripts/helpers/cql_results_helpers.js.coffee:66]: Disregard that comment: That would require that we also parse the SDEs out of non-main library HQMF

**Review by mayerm94 — APPROVED** [review]
Changes work, SDEs appear in measure view. Tests added are good

**Pull request merged**